transactions/sell_item.cdc

```diff
@@ -1,43 +1,75 @@
-import FungibleToken from "../contracts/FungibleToken.cdc"
-import NonFungibleToken from "../contracts/NonFungibleToken.cdc"
-import FlowToken from "../contracts/FlowToken.cdc"
-import ExampleNFT from "../contracts/ExampleNFT.cdc"
-import NFTStorefront from "../contracts/NFTStorefront.cdc"
+import FlowToken from 0x0ae53cb6e3f42a79
+import FungibleToken from "../contracts/utility/FungibleToken.cdc"
+import NonFungibleToken from "../contracts/utility/NonFungibleToken.cdc"
+import ExampleNFT from "../contracts/utility/ExampleNFT.cdc"
+import MetadataViews from "../contracts/utility/MetadataViews.cdc"
+import NFTStorefrontV2 from "../contracts/NFTStorefrontV2.cdc"
 
-transaction(saleItemID: UInt64, saleItemPrice: UFix64) {
-    let flowReceiver: Capability<&FlowToken.Vault{FungibleToken.Receiver}>
-    let exampleNFTProvider: Capability<&ExampleNFT.Collection{NonFungibleToken.Provider, NonFungibleToken.CollectionPublic}>
-    let storefront: &NFTStorefront.Storefront
+transaction(saleItemID: UInt64, saleItemPrice: UFix64, customID: String?, commissionAmount: UFix64, expiry: UInt64, marketplacesAddress: [Address]) {
```

Comment:
i understand commission amount to be like a 5% marketplace fee on nba top shot.

i imagine `commissionAmount` would more likely be hard-coded into the transaction in most cases - rather than being an argument subject to user input.

Reply:
I don't want to restrict so it is open for the dApps to change.